modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css

```
@@ -96,9 +101,18 @@
 		align-items: flex-end;
 		align-self: center;
 		flex-direction: column;
+		gap: 0.5rem;
 		max-width: 200px;
```

Try use bootstrap for gap,
```suggestion 
    gap-3
```